frontend/src/components/Home.vue

```diff
@@ -1,135 +1,172 @@
-                          </span>
-                          {{ menuItem.title }}
-                        </a>
-                      </li>
-                    </ul>
-                  </aside>
+
+    <section id="home">
+
+      <div class="py-6">
+        <div class="container px-6">
+          <div class="columns is-multiline is-mobile is-variable is-8-tablet is-centered is-vcentered">
+            <div v-for="item in tools" :key="item.title" class="column is-3-desktop is-6-mobile is-size-5">
+              <router-link :to="item.route">
+                <div class="card hoverable">
+                  <div class="card-image">
+                    <figure class="image is-4by3">
+                      <img :src="item.img" :alt="item.title" />
+                    </figure>
+                  </div>
+                  <footer class="card-footer has-text-centered">
+                    <a class="card-footer-item has-text-weight-bold is-size-4 is-size-5-mobile">
+                      {{ item.cardLink }}
+                    </a>
+                  </footer>
                 </div>
-                <div class="column more-padding is-v-aligned">
-                  <p class="has-text-justified" v-html="currentMenu.text"></p>
+              </router-link>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="has-background-white-ter py-6">
+        <div class="container px-6">
+          <p class="title is-size-4 has-text-centered pt-4">
+            <router-link :to="news.route" class="has-text-primary">
+              <span class="icon is-large fa-lg">
+                <i :class="`fa fa-${news.icon}`"></i>
+              </span>
+              {{ news. title }}
+            </router-link>
+          </p>
+          <div class="columns is-variable is-8 pt-3">
+            <div class="column is-6 is-size-5">
+              <p>The main versions of <b>Metabolic Atlas</b> are associated with scientific articles as follows.</p><br>
+              <div class="box">
+                <p>Version 2.0:
+                  <i>Genome-scale reconstruction of metabolic networks of model animals
+                     represents a platform for translational research</i>.
+                  Article under consideration.
+                </p>
+              </div>
+              <div class="box">
+                <p>Version 1.0: <a href="https://doi.org/10.1126/scisignal.aaz1482" target="_blank">Robinson, J., et al, 2020. <i>An atlas of human metabolism</i>. Sci. Signal. 13, eaaz1482</a></p>
+              </div>
+            </div>
+            <div class="column is-6 is-size-5">
+              <p v-html="news.text"></p>
+              <table>
+                <template v-for="el in newsItems">
+                  <!-- eslint-disable-next-line vue/valid-v-for vue/require-v-for-key -->
+                  <tr v-if="el.date">
+                    <td>{{ el.date }}</td>
```

Comment:
I'm not familiar with using dot as a separator for dates, is this an academic thing?

Reply:
I'm speaking from memory, but I think the date is used as a hash linking to the About page, so a the usual forward slash would be unusable. A dot felt also more readable than a `-`. What do you think?

Reply:
I looked at https://en.wikipedia.org/wiki/Date_format_by_country and it looks like certain countries do use `.` but `-` seems to be more widely used. You can make the call here, I have a slight preference for `-` but it's not an important issue.

Reply:
Will fix as part of MA-664.